bootstrap/bootstrapdb/database.go

```diff
@@ -22,19 +22,20 @@ type Config struct {
 
 // DB contains access to different database tables
 type DB struct {
-	kdb, ndb storage.KeyValueStore
+	kdb, ndb, adb storage.KeyValueStore
```

Comment:
Let's make a proper interface for the storage from the start rather than pushing it forward. Kademlia should be doing that as well, but we haven't gotten around to it.

I.e. ideally this should read `kademlia kademlia.DB`.

Reply:
Ok I'll make separate PR to get that in first.